internal/zinc-ivy-integration/src/main/scala/sbt/internal/inc/ZincComponentCompiler.scala

```diff
@@ -100,10 +100,7 @@ private[sbt] object ZincComponentCompiler {
       val dependencies = Vector(scalaLibrary, scalaCompiler).map(_.withConfigurations(CompileConf))
       val wrapper = dummyModule.withConfigurations(CompileConf)
       val moduleDescriptor =
-        libraryManagement.moduleDescriptor(wrapper,
-                                           dependencies,
-                                           None,
-                                           ZincLMHelper.DefaultConfigurations)
+        libraryManagement.moduleDescriptor(wrapper, dependencies, None)
```

Comment:
Component is no longer part of the configurations out of the box, so that might needs adjustment.